packages/jest-snapshot/src/__tests__/__snapshots__/printDiffOrStringified.test.ts.snap

```diff
@@ -0,0 +1,185 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`empty string expected and received single line 1`] = `
+Snapshot: <g>""</>
+Received: <r>"single line string"</>
+`;
+
+exports[`empty string received and expected multi line 1`] = `
+Snapshot: <g>"multi</>
```

Comment:
In the empty/multiline case, maybe start the multiline string on the next line so that the indentation is less off (still off by one because of the `"`)?

Reply:
Super question, as usual. Extra credit for thinking about alignment.

Here are 2 alternatives for response or maybe they provoke some other solution:

<img width="1000" alt="multiline empty" src="https://user-images.githubusercontent.com/11862657/59569275-f7c44b00-9054-11e9-8d3a-14616f97d7ba.png">

I though about omitting double quote marks, but content of string might be ambiguous with serialization of other data types.

I am thinking about backticks for multiline string as property in future data-driven diff at right:

<img width="1000" alt="multiline property value" src="https://user-images.githubusercontent.com/11862657/59569286-13c7ec80-9055-11e9-8c02-26d2d1a9dddd.png">

Reply:
Second one is probably easier on the eyes if the string itself has lines indented by 2. Wondering if inserting indentation into people's strings is a good idea though - is there a legitimate use case for copying it out of the terminal?

Reply:
Yes, inserting indentation is a concern. Although Jest in console does whether we like or not.

Of alternatives above, I agree with you to prefer at the right.

Here is the minimal solution at the left and another indented alternative at the right:

<img width="1000" alt="multi 2" src="https://user-images.githubusercontent.com/11862657/59570918-f272fa80-906c-11e9-9429-047fb9dbb3f4.png">

Unless there is a clear winner, I think to leave it alone (as it has been) for this pull request.